_data/master/navbars/reference.yml

```diff
@@ -89,15 +89,7 @@ toc:
   path: /reference/cni-plugin/configuration
 - title: Calico Kubernetes Controllers
   path: /reference/kube-controllers/configuration
-- title: Deploying on Public Cloud
-  section:
-    - title: AWS
-      path: /reference/public-cloud/aws
-    - title: Azure
-      path: /reference/public-cloud/azure
-    - title: GCE
-      path: /reference/public-cloud/gce
-- title: Deploying on Private Cloud
+- title: Network design
```

Comment:
Still showing as 'Deploying on Public Cloud' in my rendered version.

Reply:
It's probably a jekyll caching thing - I often have to `make clean` before rendering or else I get stale navbars.